test/Prototypes/Unicode.swift

```diff
@@ -63,9 +99,12 @@ final class _StringStorage<Element: UnsignedInteger>
       body(UnsafeBufferPointer(start: UnsafePointer($0.baseAddress), count: count))
     }
   }
-/*}
+/*
+}
+
+extension _StringStorage : _NSStringCore {
+*/
 
-extension _StringStorage : _NSStringCore {*/
```

Comment:
The compiler error is:

@objc is not supported within extensions of generic classes

Is this a known limitation, or should I open a new JIRA?